libs/shared/npm/src/lib/workspace-dependencies.ts

```diff
@@ -80,13 +81,16 @@ export async function workspaceDependencyPath(
 export function importWorkspaceDependency<T>(
   importPath: string,
   logger?: Logger,
+  workspacePath: string = __dirname,
 ): Promise<T> {
   if (platform() === 'win32') {
     importPath = importPath.replace(/\\/g, '/');
   }
 
+  const workspaceRequire = Module.createRequire(workspacePath);
+
   // eslint-disable-next-line @typescript-eslint/no-var-requires
-  const imported = require(importPath);
+  const imported = workspaceRequire(importPath);
```

Comment:
the webpack bundler in `nx run nxls:package` will replace this require to a webpack require that always errors. using module and createRequire is one of a few solutions. (imo, this would take a larger refactor to make use of the `workspacePath` argument, but it could simplify a lot of logic)

Reply:
are you saying that this always errors? or only for helix?
This is a widely used function so I'm hesitant to change it and I wonder why this is causing issues in helix but not otherwise...

Reply:
okay thanks for the investigation @forivall
Tbh I'd rather switch away from webpack to esbuild and align the builds between build/package than start patching up the webpack config and adding a new dependency and everything. 

Is this something you'd be willing to do in this PR? Or should I try the migration and then you can build on top of that?

Reply:
yeah, the build already does most of what package does, so i think it would just be a noop with a dependency on `build`, and fix the assets and other configurations in the `build` output to get the missing `bin` script to copy, etc.